server/server/app.py

```diff
@@ -0,0 +1,57 @@
+from flask import Flask, redirect, url_for
+import json
+from server.server.Utils import Connection, and_cond
+from server.server.models import Products, CartItem, User
+import flask_login
+app = Flask(__name__)
+
+conn = Connection(local=False)
+
+
+def extract_dict_from_query(*query):
+    ret = []
+    for item in query:
+        item_dict = item.__dict__
+        item_dict.pop("_sa_instance_state")
+        ret.append(item_dict)
+
+    return ret
+
+
+def extract_dict_from_query_list(query_list):
+    return extract_dict_from_query(*query_list)
```

Comment:
I believe you don't need both `extract_dict_from_query_list` and `extract_dict_from_query` since `*query_list` is good also for a single query.

Reply:
the problem is that *query handles func(arg1,arg2.arg3.....) but not func([arg1,arg2,arg3....]),
this is why we say from list*